dascore/proc/correlate.py

```diff
@@ -147,15 +147,20 @@ def correlate(
     ... )
     >>>
     >>> # Example 3
-    >>> # Use 2nd channel (python is 0 indexed) along distance as master channel
-    >>> cc_patch = patch.correlate(distance=1, samples=True)
+    >>> # Correlate along distance having the 2nd channel (python is 0 indexed)
+    >>> # as a master channel and skip every other channel to reduce memory
+    >>> cc_patch = (
+    ...     patch.decimate(distance=2, filter_type=None)
```

Comment:
Just a minor thing but decimating along distance by a factor of 2 results in a patch with a half number of channels. So, having kwargs of `distance=1` basically means correlating the third channel of the original patch before decimation with other channels.